conans/client/conan_api.py

```diff
@@ -421,24 +421,23 @@ def export_pkg(self, conanfile_path, name, channel, source_folder=None, build_fo
             raise
 
     @api_method
-    def download(self, reference, remote_name=None, package=None, recipe=False):
-        # FIXME: The "package" parameter name is very bad, it is a list of package_ids
-        if package and recipe:
-            raise ConanException("recipe parameter cannot be used together with package")
+    def download(self, reference, remote_name=None, package_list=None, recipe=False):
```

Comment:
I think we need to unify the name here of `package_list`. I think `packages` is the name used in other api methods